src/lib/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js

```diff
@@ -177,6 +179,10 @@ class LoansListEntry extends Component {
           {DateTime.fromISO(endDate).toLocaleString()}
         </Label.Detail>
       </Label>
+      <Label basic>
+        <Icon className={delivery === 'PICKUP' ? 'university' : 'dolly'} />{' '}
+        {invenioConfig.CIRCULATION.deliveryMethods[delivery]}
```

Comment:
Minor: could you wrap it in `<Label.Detail>` ? the text is a bit too bold

Reply:
@KonstantinaStoikou sorry for this, it was me. But after the last demo, it looks like that important text is not very visible in the profile page and user could get confused, at least this was the feedback we got.
The idea is to highlight very very much the loan return date, and highlight less the rest.